src/postamble.js

```diff
@@ -406,9 +406,9 @@ function abort(what) {
   if (ENVIRONMENT_IS_PTHREAD) console.error('Pthread aborting at ' + new Error().stack);
 #endif
   if (what !== undefined) {
+    what += '';
```

Comment:
What is this doing?   Coercing to a string?  Surly there is a better way?  `what = what.toString()`?

Reply:
That works for some things but not others. For example a number has no `.toString()` method on it, unlike objects. `x + ''` will always coerce to a string no matter what `x` is.